src/pages/saksbehandling/søknadsbehandling/formue/Formue.tsx

```diff
@@ -199,11 +206,22 @@ const Formue = (props: {
         form.trigger('epsFormue.depositumskonto');
     }, [watch.epsFormue?.innskuddsbeløp]);
 
+    useEffect(() => {
+        hentSkattemeldingBruker({ fnr: props.søker.fnr });
+    }, []);
+
     useEffect(() => {
         if (watch.epsFnr && watch.epsFnr.length === 11) {
-            fetchEps(watch.epsFnr);
+            fetchEps(watch.epsFnr, (res) => {
+                if (skattemeldingToggle) {
+                    hentSkattemeldingEPS({ fnr: res.fnr });
+                }
+            });
         } else {
-            resetEpsToInitial();
+            if (skattemeldingToggle) {
+                resetEpsToInitial();
```

Comment:
Kunne toggle av vært en if ytterst i use-effect og bare early returne her? Så trenger vi ikke to ifs for skattemeldingtoggle

Reply:
Hehe ser nå att denne bli litt feil. Det er `resetSkattemeldingEPS` som skal vara wrappet i feature toggle här.  Men ja, hvis jag skjønte det riktigt så går det å flytte `resetSkattemeldingEPS` til starten av useEffecten, og kvitte seg med en if-settning. 😄